### DISCUSSION TITLE
Possible Inconsistency between image and guide

### DISCUSSION THREAD

**Discussion by chadoulis** [STELLA-Landsat/STELLA#17] [Build Notes: How do I make a STELLA?]
In [Build Steps - Part 2 Header Pins](https://landsat.gsfc.nasa.gov/stella/stella-two/stella-two-build-instructions/#Wiring), the image depicts the last group of pins in the position 28-35 D while the guide suggests pinning it in the position 28-35 B. Which one of the two is the correct one?

**Comment by paulmirel** [comment]
28 to 35 D is correct.
